src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -641,8 +641,8 @@ private Path discoverSdkPath() {
               + ": exception thrown when searching for Google Cloud SDK", ex);
         }
       }
-      throw new AppEngineException("Google Cloud SDK path was not provided and could not be"
-          + " found in any known install locations.");
+      throw new CloudSdkNotFoundException("The Google Cloud SDK could not be found in the default"
+          + " locations and no path was provided.");
```

Comment:
"customary" works for me.  I was unsure about _no path was provided_ but that seems covered with `GOOGLE_CLOUD_SDK_HOME` and the preferences.